packages/rich-text-editor/docs/EditableRichTextContent.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react"
+import {
+  EditableRichTextContent,
+  RichTextEditor,
+  EditorContentArray,
+} from "@kaizen/rich-text-editor"
+import { Button } from "@kaizen/button"
+import { Box } from "@kaizen/component-library"
+import { CATEGORIES, SUB_CATEGORIES } from "../../../storybook/constants"
+import dummyContent from "./dummyContent.json"
+
+export default {
+  title: `${CATEGORIES.components}/${SUB_CATEGORIES.richTextEditor}/Editable Rich Text Content`,
+  component: EditableRichTextContent,
+  parameters: {
+    docs: {
+      description: {
+        component:
+          'import { EditableRichTextContent } from "@kaizen/rich-text-editor"',
+      },
+    },
+  },
+}
+
+export const EditableRichTextContentStory = args => <InlineEditor {...args} />
+EditableRichTextContentStory.storyName = "Default"
+EditableRichTextContentStory.args = {
+  content: dummyContent,
+  labelText: "Shared notes",
+  isLabelHidden: false,
+}
+
+function InlineEditor(props: {
+  content: EditorContentArray
+  labelText: string
+}) {
+  const [editMode, setEditMode] = useState<boolean>(false)
+  const [rteData, setRTEData] = useState<EditorContentArray>(dummyContent)
+  const handleContentClick = () => setEditMode(true)
+  const handleCancel = () => setEditMode(false)
+
+  if (editMode) {
+    return (
+      <>
+        <RichTextEditor
+          labelText={props.labelText}
+          controls={[
+            { name: "bold", group: "inline" },
+            { name: "italic", group: "inline" },
+            { name: "underline", group: "inline" },
+            { name: "orderedList", group: "list" },
+            { name: "bulletList", group: "list" },
+            { name: "link", group: "link" },
+          ]}
+          value={rteData}
+          onChange={data => setRTEData(data)}
+        />
+        <Box mt={0.5} style={{ display: "flex", justifyContent: "end" }}>
```

Comment:
Don't know if this is required since its just a story example but thought I'd just call out this minor spacing nit
![Screen Shot 2022-07-13 at 10 33 18 am](https://user-images.githubusercontent.com/36558508/178623842-1236bd96-f445-4791-b8be-64acb75d7e46.png)

Reply:
Yeah got a little lazy here